go/pkg/client/cas_upload.go

```diff
@@ -287,6 +287,9 @@ func (c *Client) uploadUnified(ctx context.Context, entries ...*uploadinfo.Entry
 			contextmd.Infof(ctx, log.Level(2), "Skipping upload of empty entry %s", ue.Digest)
 			continue
 		}
+		if ue.IsVirtualInput && ue.IsFile() {
+			return nil, 0, fmt.Errorf("virtual input with digest: %s provided, but does not exist in CAS", ue.Digest)
+		}
```

Comment:
This block is not clear to me. Why is it an error to have a missing virtual input?

Reply:
VirtualInputs should exist on RBE and if it doesn't then it means something likely already went wrong before reaching this step. I don't think we should be uploading a possibly corrupted digest in this case.